apps/Xtreme Download Manager/install

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+"${DIRECTORY}/pkg-install" "openjdk-11-jdk ffmpeg youtube-dl" "$(dirname "$0")" || error "Failed installing openjdk!"
+
+cd $HOME
+mkdir -p xdm 
+cd xdm
+wget -O xdm.tar.xz https://github.com/subhra74/xdm/releases/download/7.2.11/xdm-setup-7.2.11.tar.xz || error "Failed to download XDM setup tar file!"
+tar -xf xdm.tar.xz || error "Failed to extract XDM setup tar file!"
+sudo ./install.sh | grep -v "Installation completed" || error "Failed to execute XDM setup script!"
+
+# Use system java
+sudo sed -i s=/opt/xdman/jre/bin/java=java=g /opt/xdman/xdman || error "Failed to change to system Java runtime in /opt/xdman/xdman!"
```

Comment:
Does this work without this? they probably bundled java because it's the version that works. so unless there is no arm version (probably the case) we should use the included java.

Reply:
No, using the bundled Java will get `Exec format error` as it is for x64 only. 
Here is where I got the solution: https://github.com/subhra74/xdm/issues/293